README.md

```diff
@@ -56,21 +56,21 @@ CI runs a series of steps;  this the sequence to do it locally, along with some
 1. **Pull down the latest docker containers**
 
     ```
-    docker-compose pull
+    docker compose pull
     ```
 
 2. **Start up the docker services needed for testing**
 
     Once everything has been successfully pulled, start up the docker services needed for testing (all but the web container)
 
     ```
-    docker-compose up -d sdr-api techmd
+    docker compose up -d sdr-api techmd dor-indexing-workers
```

Comment:
This is the shortest way to ask Argo's `docker compose` configuration to spin up all services *except* the web container.

Reply:
@ndushay https://github.com/sul-dlss/argo/pull/3702 is where that service was added. This change only modifies the README to have a more complete way to spin up services.